docs/user/admin/integrations/supported-integrations.md

```diff
@@ -31,29 +29,43 @@ include action buttons to create and merge PRs and add labels, reviewers, and co
 * BitBucket Cloud
 * GitLab
 
-#### Issue Tracking Integrations
+## Issue Tracking Integrations
 
 Atomist receives events for issue and issue comment creation and update.
 
 * GitHub Issues
 
-#### CI Integrations
+## CI Integrations
 
-Atomist can receive build notifications from:
+One of the tools you are most likely to integrate is Continuous Integration (CI). Atomist can receive build notifications from:
 
 * Jenkins
 * Travis CI
 * TeamCity
 * Circle CI
 * any other build system, as a POST to our webhook
 
-### Running other programs from Atomist
+Integrating a custom CI tool with Atomist is simple. Simply invoke Atomist
+hooks to send events around build and artifact creation.
+
+If integrating CI tools, we recommend the following:
+
+- CI tools are great for building and generating artifacts. They are
+  often abused as a PaaS for `bash`. If you find your CI usage has
+  you programming in `bash` or YML, consider whether invoking such
+  operations from Atomist event handlers might be a better model.
+- Use Atomist generators to create your CI files, and Atomist
+  editors to keep them in synch, minimizing inconsistency.
+
+## Running other programs from Atomist
 
 In response to events, you can trigger actions on other systems from your Atomist automations.
-Some of the ones already implemented include:
+Some of the ones we've already implemented include:
 
-* Checkstyle
+* [Checkstyle](/pack/checkstyle/)
 * TSLint
```

Comment:
I couldn't find a link to this either as a package or a repo, so I hope leaving this blank is ok!